ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -184,29 +185,37 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
         </TableCell>
 
         <TableCell>
+          <Button
+            className={classes.linksButton}
+            variant="outlined"
+            component={Link}
+            to={`/provisioners/${encodeURIComponent(
```

Comment:
(this is unrelated to the PR, but from git history I see this change is yours. We don't have provisioners and worker-types anymore, should they be in the url 🤔 should we create a bug to change the url?)

Reply:
This is the old "provisioners" stuff.  Once the new worker-manager list-of-workers UI is finished, I think well get rid of (or redirect from) `/provisioners`.

Reply:
> > It didn't display errors from my table tho (I ran this against my cluster). Is it in accord with the master-plan?
> 
> It is if there weren't any errors to display. Otherwise there's a bug! I tested it against the current production deployment, and a worker pool that I knew had some errors.

oh, worker manager is in production already? I didn't know that.

I do have an error in my cluster. In the Network tab, I didn't even see the request being made. Lemme try again against the production. It's `https://taskcluster.net`, right?